spaceship.zsh

```diff
@@ -88,6 +94,55 @@ spaceship_time() {
   echo -n "%{$reset_color%} "
 }
 
+# Battery
+spaceship_battery(){
+  [[ $SPACESHIP_BATTERY_SHOW == false ]] && return
+
+  local os=$(uname);
+  local battery_status="";
+  local battery_percent="0";
+
+  if [[ "$os" = "Linux" ]]; then
+    if which upower > /dev/null ; then
+      local battery_power=$(upower -i $(upower -e | grep BAT | tail -1) | grep state | awk '{print $2}');
+      if [[ $battery_power = "charging" ]]; then
+        battery_status=$SPACESHIP_BATTERY_CHARGING;
+      elif [[ $battery_power = "discharging" ]]; then
+        battery_status=$SPACESHIP_BATTERY_DISCHARGING;
+      elif [[ $battery_power = "fully-charged" ]]; then
+        battery_status=$SPACESHIP_BATTERY_FULL;
+      fi
+      battery_percent=$(upower -i $(upower -e | grep BAT | tail -1) | grep percentage | awk '{print $2}' | sed "s|%||g");
+    fi
+  fi
+
+  if [[ "$os" = "Darwin" ]]; then
+    local battery_power=$(pmset -g batt | tail -1 | awk '{print $4}' | tr -d "%;");
```

Comment:
With the added id details in pmset, Isn't it be `awk '{print $3}'` ?

![pmset args](https://i.imgur.com/CW6G7pu.png)